share/spack/docker/ubuntu-1604.dockerfile

```diff
@@ -17,16 +17,23 @@ COPY share $SPACK_ROOT/share
 COPY var   $SPACK_ROOT/var
 RUN mkdir -p $SPACK_ROOT/opt/spack
 
-RUN apt-get -yqq update                                   \
- && apt-get -yqq install                                  \
-        build-essential ca-certificates curl       g++    \
-        gcc             gfortran        git        gnupg2 \
-        iproute2        lmod            lua-posix  make   \
-        openssh-server  python          python-pip tcl    \
-        unzip                                             \
- && pip install boto3                                     \
+RUN apt-get -yqq update                                        \
+ && apt-get -yqq install                                       \
+        build-essential ca-certificates curl       g++         \
+        gcc             gfortran        git        gnupg2      \
+        iproute2        lmod            locales    lua-posix   \
+        make            openssh-server  python     python-pip  \
+        python3-pip     tcl             unzip                  \
+ && locale-gen en_US.UTF-8                                     \
+ && pip install boto3                                          \
+ && pip3 install boto3                                         \
```

Comment:
I think we can drop python2 in favor of python3.  No need to have both in the same image.

Reply:
Done